src/comments/comments.service.ts

```diff
@@ -1,36 +1,39 @@
-import { Injectable } from '@nestjs/common';
+import { forwardRef, Inject, Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
-import { InfoArticle } from 'src/articles/dto/info-article.dto';
+import { ArticlesService } from 'src/articles/articles.service';
+import { User } from 'src/user/entities/user.entity';
 import { Repository } from 'typeorm';
+
 import { CreateCommentDto } from './dto/create-comment.dto';
-import { UpdateCommentDto } from './dto/update-comment.dto';
+
 import { Comments } from './entities/comment.entity';
 
 @Injectable()
 export class CommentsService {
-  @InjectRepository(Comments) private commentRepository: Repository<Comments>;
-  create(createCommentDto: CreateCommentDto, idArticle: number) {
-    // const comment = this.commentRepository.create({
-    //   ...createCommentDto,
-    //   idArticle,
-    // });
-    // return this.commentRepository.save(comment);
-    return 0;
-  }
-
-  findAll() {
-    return `This action returns all comments`;
+  constructor(
+    @InjectRepository(Comments) private commentRepository: Repository<Comments>,
+    private articleService: ArticlesService,
+  ) {}
+  async create(user: User, comments: CreateCommentDto, articleId: number) {
+    const createComment = this.commentRepository.create(comments);
```

Comment:
nên create comment với user cùng 1 lần
this.commentRepository.create({...comments, author: user})